fitbenchmarking/cli/main.py

```diff
@@ -138,8 +138,10 @@ def run(problem_sets, options_file='', debug=False):
     root = os.path.dirname(inspect.getfile(fitbenchmarking))
     template_dir = os.path.join(root, 'templates')
     env = Environment(loader=FileSystemLoader(template_dir))
-    style_css = os.path.join(template_dir, 'main_style.css')
-    custom_style = os.path.join(template_dir, 'custom_style.css')
+    local_css_dir = os.path.join(options.results_dir,'css')
```

Comment:
These four lines of code look almost identical each time we create a html page, I suggest this is put into `utils` or something like that to avoid duplication

Reply:
I can only pull out two -- the  particular css files linked to depend on the page being generated.  Have created a function to do this, which I think makes the code a bit more understandable.